modules/gcs/variables.tf

```diff
@@ -89,3 +89,9 @@ variable "retention_policies" {
   type        = map(map(string))
   default     = {}
 }
+
+variable "logging" {
+  description = "Per-bucket logging."
+  type        = map(map(string))
```

Comment:
Can you narrow down the type? Probably

```hcl
type = map(object({
  log_bucket = string
  log_object_prefix = string
}))
```

I would also rename the variable to `logging_config` for conformity with other modules.

Reply:
Renamed.